<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotorsSensorsWidget</class>
 <widget class="QWidget" name="MotorsSensorsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Motors and Sensors</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="motors">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>70</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>33</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Motors</string>
         </property>
         <property name="icon">
          <iconset resource="../rc/icons.qrc">
           <normaloff>:/icons/car.png</normaloff>:/icons/car.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="servos">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>70</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>33</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Servos</string>
         </property>
         <property name="icon">
          <iconset resource="../rc/icons.qrc">
           <normaloff>:/icons/cogs.png</normaloff>:/icons/cogs.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Create3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>70</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>33</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Create 3</string>
         </property>
         <property name="icon">
          <iconset resource="../rc/icons.qrc">
           <normaloff>:/icons/robot.png</normaloff>:/icons/robot.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="sensorGraph">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>70</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>33</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Sensor Graph</string>
         </property>
         <property name="icon">
          <iconset resource="../rc/icons.qrc">
           <normaloff>:/icons/chart-line.png</normaloff>:/icons/chart-line.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sensorList">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>70</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>33</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Sensor List</string>
         </property>
         <property name="icon">
          <iconset resource="../rc/icons.qrc">
           <normaloff>:/icons/list-alt.png</normaloff>:/icons/list-alt.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="camera">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>70</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>33</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Camera</string>
         </property>
         <property name="icon">
          <iconset resource="../rc/icons.qrc">
           <normaloff>:/icons/camera-retro.png</normaloff>:/icons/camera-retro.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>180</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Tello">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16770000</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>33</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tello</string>
       </property>
       <property name="icon">
        <iconset resource="../rc/icons.qrc">
         <normaloff>:/icons/AerialLogo.png</normaloff>:/icons/AerialLogo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>45</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rc/construction.qrc"/>
  <include location="../rc/icons.qrc"/>
 </resources>
 <connections/>
</ui>
